client.go

```diff
@@ -83,7 +86,7 @@ func NewClientWithURL(apiKey, civoAPIURL, region string) (*Client, error) {
 
 	client := &Client{
 		BaseURL:   parsedURL,
-		UserAgent: "civogo/" + Version,
+		UserAgent: "civogo/" + utils.GetVersion(),
```

Comment:
This is confusing. We're supposed to set `client.UserAgent := "civogo/"+utils.GetVersion()` and then `c.SetUserAgent(&component{})` separately?

Reply:
Will keep the civogo version in every request